Core/src/main/java/com/plotsquared/core/generator/HybridPlotWorld.java

```diff
@@ -462,13 +506,22 @@ public void addOverlayBlock(short x, short y, short z, BaseBlock id, boolean rot
         if (y >= height) {
             if (y > lastOverlayHeightError) {
                 lastOverlayHeightError = y;
-                LOGGER.error(String.format("Error adding overlay block. `y > height`. y=%s, height=%s", y, height));
+                LOGGER.error(String.format(
+                        "Error adding overlay block in world %s. `y > height`. y=%s, height=%s",
```

Comment:
If we already touch that, we could avoid using String.format and use the logger functionality directly (using `{}` for arguments)